### ISSUE TITLE
Unexpected error in map keys

### ISSUE THREAD

**Issue by akiym** [anboralabs/intellij-firebase-highlighter#108]
**Describe the bug**
Unexpected error occurs in the second and subsequent map keys.

**To Reproduce**
```
rules_version = '2';
service cloud.firestore {
  match /databases/{database}/documents {
    allow read, write: if false;

    function test() {
      let a = {'a': 1, 'b': 2, 'c': 3};
      return a;
    }
  }
}
```

**Expected behavior**
No unexpected error occurs in maps keys.

**Screenshots**
<img width="469" alt="Screen Shot 2022-12-19 at 15 06 36" src="https://user-images.githubusercontent.com/76525/208358912-e5a9b42d-fc4a-4b44-b8ab-2f3f0e1b16b9.png">

**Desktop (please complete the following information):**
 - OS: macOS 12.6.2
 - Editor: IntelliJ IDEA 2022.3 
 - Plugin Version: 2022.5.3

**Comment by dalgarins** [comment]
Issue cloned to https://github.com/anboralabs/intellij-firebase/issues/13

**Comment by dalgarins** [comment]
<img width="907" alt="Captura de Pantalla 2022-12-19 a la(s) 12 16 14 p m" src="https://user-images.githubusercontent.com/8527174/208482744-b8536e79-a444-4cc9-afc9-54d9d9d7f719.png">

@akiym the issue will be fixed on version: 2022.5.4

**Comment by akiym** [comment]
I've confirmed it works correctly with 2022.5.4. Thank you for fixing so quickly!
